jsonnetfile.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I don't think we need either of these dependencies for this library.
As we only need to generate graph and single stat panels, we should be fine without these. :wink: :blush: